TwitchPlugin/TwitchPlugin/TwitchPlugin.csproj

```diff
@@ -1,328 +1,213 @@
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <Nullable>warnings</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Authors>Logitech Europe SA</Authors>
+    <Copyright>Copyright © 2024 Logitech Europe SA. All rights reserved.</Copyright>
+    <AssemblyVersion>5.10.4</AssemblyVersion>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+    <BaseOutputPath>$(SolutionDir)bin\</BaseOutputPath>
+    <OutputPath Condition="$(OS) == 'Windows_NT'">$(BaseOutputPath)$(Configuration)\win\</OutputPath>
+    <OutputPath Condition="$(OS) != 'Windows_NT'">$(BaseOutputPath)$(Configuration)/mac/</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject />
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="PluginApi">
+      <HintPath>$(SolutionDir)TwitchPlugin\TwitchPlugin\Sdk\net8\PluginApi.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Actions\ShieldModeToggleCmd.cs" />
+    <Compile Include="Actions\RunCommercialCmd.cs" />
+    <Compile Include="Actions\ChatFollowersCommand.cs" />
+    <Compile Include="Actions\ChatSlowModeCommand.cs" />
+    <Compile Include="Actions\ClearChatCommand.cs" />
+    <Compile Include="Actions\CreateClipCommand.cs" />
+    <Compile Include="Actions\CreateMarkerCommand.cs" />
+    <Compile Include="Actions\UniversalStateSwitch.cs" />
+    <Compile Include="Actions\ChatSubToggleCmd.cs" />
+    <Compile Include="Actions\GenericOnOffSwitch.cs" />
+    <Compile Include="Actions\ChatEmoToggleCmd.cs" />
+    <Compile Include="Actions\InternalDebugCommand.cs" />
+    <Compile Include="Actions\ViewerCountCommand.cs" />
+    <Compile Include="Authenticator\AuthenticationServer.cs" />
+    <Compile Include="DisplayNameMaps.cs" />
+    <Compile Include="Exceptions\InvalidAccessTokenException.cs" />
+    <Compile Include="Localization\Localization.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Localization.resx</DependentUpon>
+    </Compile>
+    <Compile Include="Logging\Logger.cs" />
+    <Compile Include="Models\ActionHelpers.cs" />
+    <Compile Include="Models\TwitchPluginConfig.cs" />
+    <Compile Include="Models\EventArgs.cs" />
+    <Compile Include="Models\UserInfo.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="TwitchApplication.cs" />
+    <Compile Include="TwitchPlugin.cs" />
+    <Compile Include="Proxy\TwitchProxy.Channel.cs" />
+    <Compile Include="Proxy\TwitchProxy.Connectivity.cs" />
+    <Compile Include="Proxy\TwitchProxy.cs" />
+    <None Include="Proxy\TwitchProxy.Chatters.cs" />
+    <Compile Include="Proxy\TwitchProxy.Viewers.cs" />
+  </ItemGroup>
+  <ItemGroup>

```

Comment:
Looks like we have a bunch of duplicates here.

Reply:
The duplicates were removed